index.bs

```diff
@@ -696,6 +698,38 @@ authorizing an authenticator with which to complete the operation.
 </div>
 
 
+### Inform user platform authenticator capability - isPlatformAuthenticatorReady() method ### {#isPlatformAuthenticatorReady}
+
+<div link-for-hint="WebAuthentication/isPlatformAuthenticatorReady">
+This method can be used by [RPS] to inform user they can create a new credential with one or more platform authenticators.
+The method gives [RPS] the ability to detect whether the client is capable of registering credentials with 
```

Comment:
AIUI it only gives the RP the ability to tell the client that it (the RP) supports use of WebAuthn. The RP does not get to detect anything.